components/setting_picture.tsx

```diff
@@ -15,35 +14,48 @@ import OverlayTrigger from 'components/overlay_trigger';
 import Tooltip from 'components/tooltip';
 import LoadingWrapper from 'components/widgets/loading/loading_wrapper';
 
-export default class SettingPicture extends Component {
+type Props = {
+    clientError?: ReactNode;
+    serverError?: ReactNode;
+    src?: string | null;
+    defaultImageSrc?: string;
+    file?: File | null;
```

Comment:
just curious will they be null ? is the tsc complaining if we dont put the type as null. Because being `?:` undefined should cover that.